src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -418,6 +422,55 @@ class WorkspaceService(dataSource: DataSource, workspaceDAO: WorkspaceDAO, entit
       // to ensure it is evaluated within the transaction
       RequestComplete(methodConfigurationDAO.list(workspaceNamespace, workspaceName, txn).toList)
     }
+
+  // TODO: replace this stub with the actual expression parser
+  private def resolveInputs( methodConfig: MethodConfiguration, entity: Entity ): String =
+    """{"hello": "world"}"""
+
+  def submitJob(jobDesc: JobDescription, authCookie: HttpCookie): PerRequestMessage =
+    dataSource inTransaction { txn =>
+      val wsNamespace = jobDesc.methodName.workspaceName.namespace
+      val wsName = jobDesc.methodName.workspaceName.name
+      val mcNamespace = jobDesc.methodName.namespace
+      val mcName = jobDesc.methodName.name
+      methodConfigurationDAO.get(wsNamespace, wsName, mcNamespace, mcName, txn) match {
+        case None =>
+          RequestComplete(StatusCodes.NotFound,
+            "No method configuration named "+mcNamespace+"."+mcName+" in workspace "+wsNamespace+"."+wsName)
```

Comment:
forward slash has more commonly been used between namespace and name as opposed to a period.

also scala's string formatter is nice:
s"No method configuration named ${mcNamespace}/${mcName} in workspace ${wsNamespace}/${wsName}"